controler/usersControler.js

```diff
@@ -0,0 +1,43 @@
+const peopleModel = require("../models/People");
+const getUsers = (req, res, next) => {
+  res.render("users");
+};
+
+const createUser = async (req, res, next) => {
+  try {
+    const peopleInfo = await peopleModel.find();
+    if(peopleInfo== []){
```

Comment:
hope that promise will return an array then perfect way to check the empty state is `peoplenfo.length == 0`